doc/source/rest.j2

```diff
@@ -10,6 +10,13 @@ provide an `Authorization` header in your HTTP requests with a valid username
 (the password is not used). The "admin" username is granted all privileges,
 whereas any other username is recognize as having standard permissions.
 
+.. note::
+
+   The username should be encoded in Base64 when using `basic` mode as per
```

Comment:
If you want to paraphrase the standard, I don't think this is enough. It's `username:password` that must be encoded, except the password is not used (yet), so it'd be just `admin:` that needs to be encoded. :)

I'd prefer we make just it clear that the `basic` authentication mechanism of Gnocchi is actually a `Basic` authentication is the sense of HTTP.